src/textbooks/textbook-card/TextbooksCard.jsx

```diff
@@ -109,7 +114,7 @@ const TextbookCard = ({
         close={closeDeleteModal}
         title={intl.formatMessage(messages.deleteModalTitle, { textbookTitle: textbook.tabTitle })}
         description={intl.formatMessage(messages.deleteModalDescription)}
-        onDeleteSubmit={() => onDeleteSubmit(textbook.id)}
+        onDeleteSubmit={() => handleDeleteButtonSubmit(textbook.id)}
```

Comment:
[question] Can we change to 
`onDeleteSubmit={handleDeleteButtonSubmit}`

Reply:
Fixed